src/Command/StoreMigrateCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\LaravelEventSourcing\Command;
+
+use Patchlevel\EventSourcing\Console\InputHelper;
+use Patchlevel\EventSourcing\Console\OutputStyle;
+use Patchlevel\EventSourcing\Message\Pipe;
+use Patchlevel\EventSourcing\Message\Translator\Translator;
+use Patchlevel\EventSourcing\Store\Store;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use function count;
+
+#[AsCommand(
+    'event-sourcing:store:migrate',
+    'migrate events from one store to another',
+)]
+final class StoreMigrateCommand extends Command
```

Comment:
I think this should to the library in 4.x so that we dont need to maintain it twice or more (here and symfony-bundle)